src/synopsys-action/tools-parameter.ts

```diff
@@ -304,7 +304,7 @@ export class SynopsysToolsParameter {
     }
 
     // This condition is required as per ts-lint as these fields may have undefined as well
-    if (githubRepoName != null && githubBranchName != null && githubRepoOwner != null) {
+    if (githubRepoName != null && githubBranchName != null && githubRepoOwner != null && githubHostUrl != null) {
```

Comment:
if githubHostUrl is empty, other github data will not set. It will be empty in case of github cloud.